src/config/index.js

```diff
@@ -24,57 +26,67 @@ function createConfig () {
     },
     config: {
       doc: `
-    The name of a config file in the 'config' directory.
-    e.g. 'default.json'
-  `,
+        The name of a config file in the 'config' directory.
+        e.g. 'default.json'
+      `,
       format: String,
       default: 'default.json',
       arg: 'config'
     },
     server: {
       port: {
         doc: `
-      The port to be used when the server starts running.
-    `,
+          The port to be used when the server starts running.
+        `,
         format: 'port',
         default: 3000,
         env: 'PORT'
       },
       prefix: {
         doc: `
-      The API path used to get tasks, so for example if you wanted
-      to get a listing of all the tasks the application knows about,
-      it would be at /tasks.
-    `,
+          The API path used to get tasks, so for example if you wanted
+          to get a listing of all the tasks the application knows about,
+          it would be at /tasks.
+        `,
         format: 'root-relative-path',
         default: '/'
+      },
+      corsAllowedOrigins: {
+        doc: `
+          A list of values the server will provide in Access-Control-Allow-Origin
+          headers. This allows the client to be hosted on a different domain
+          from the API while still allowing communication between the two.
+        `,
+        format: 'string-list',
+        default: [],
+        env: 'CORS_ALLOWED_ORIGINS'
```

Comment:
You might be able to use https://github.com/mozilla/node-convict/blob/master/packages/convict/test/cases/array.js#L1-L8

Reply:
I tried this initially, but thought that since this particular value will be unique to each organization, I felt like keeping it in an environment variable as a string would be the most flexible solution. I don't see that the Array type in convict works for environment variables though.

Reply:
Hm - I guess it looks like convict only does strings for env vars - not really worth messing with it, so I think we should go ahead and merge this for now